src/MSBuild.Conversion.SDK/TargetFrameworkHelper.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Text.Json;
+
+using Microsoft.Build.Locator;
+
+namespace MSBuild.Conversion.SDK
+{
+    public static class TargetFrameworkHelper
+    {
+        public static string FindHighestInstalledTargetFramework(bool? allowPreviews)
+        {
+            var usePreviewSDK = allowPreviews == true;
+
+            // Finds SDK path
+            string sdkPath = null;
+            try
+            {
+                sdkPath = Path.GetFullPath(Path.Combine(MSBuildLocator.QueryVisualStudioInstances().Single().VisualStudioRootPath, "..", ".."));
```

Comment:
Note: to support a future X to M, this may need to be a little more intricate. But out of scope for this pr

Reply:
I do not know what this means.